solr/solrj/src/java/org/noggit/JSONUtil.java

```diff
@@ -17,11 +17,19 @@
 package org.noggit;
 
 public class JSONUtil {
+  @SuppressWarnings("MutablePublicArray")
   public static final char[] TRUE_CHARS = new char[] {'t', 'r', 'u', 'e'};
+
+  @SuppressWarnings("MutablePublicArray")
   public static final char[] FALSE_CHARS = new char[] {'f', 'a', 'l', 's', 'e'};
+
+  @SuppressWarnings("MutablePublicArray")
   public static final char[] NULL_CHARS = new char[] {'n', 'u', 'l', 'l'};
+
+  @SuppressWarnings("MutablePublicArray")
   public static final char[] HEX_CHARS =
       new char[] {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
+
   public static final char VALUE_SEPARATOR = ',';
   public static final char NAME_SEPARATOR = ':';
   public static final char OBJECT_START = '{';
```

Comment:
I think we can change the access modifiers to be package-protected.  These are internals that should not be public.  Perhaps we will need the SuppressWarnings annotation but lets do this little thing now too please?

Reply:
addressed and removed the suppresswarnings since it wasn't needed anymore.